### DISCUSSION TITLE
Failed to fetch dynamically imported module

### DISCUSSION THREAD

**Discussion by grantholle** [inertiajs/inertia#1352] [Help]
I've found that after I deploy and users are still in the app, they start getting "Failed to fetch dynamically imported module" errors because the hash in the filename gets changed. [This post](https://stackoverflow.com/questions/72376333/failed-to-fetch-dynamically-imported-module) describes the issue in the last update. I need the cache-busting help with the hash, but also don't want a flood of those errors coming in...

Is there a way we can catch that error and just reload the page? Is that something I can put in the `resolve` inside of the `createInertiaApp()` call?

I know this is a combination of Vite + Inertia, but since Inertia is what resolves the component I came here first :smile:

**Comment by devgraduan** [comment]
I handle this issue with https://vite.dev/guide/build#load-error-handling

```
window.addEventListener('vite:preloadError', (event) => {
  window.location.reload() // for example, refresh the page
})
```
